pkg/defaults/policies/files/curl-cve.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "db79755e-6e03-40f5-b914-b0bee9e1c20b",
+  "name": "Curl Buffer Overflow Vulnerabilities",
+  "description": "Alert on deployments with images containing severe curl and libcurl vulnerabilities (CVE-2023-38545 and CVE-2023-38546). There are flaws in curl command line tool and libcurl library in versions from 7.69.0 to 8.3.0.",
```

Comment:
I read that `CVE-2023-38546` is different. It is about cookie injection, not buffer overflow. See https://curl.se/docs/CVE-2023-38546.html

`CVE-2023-38545` is the one with buffer overflow and it affects both `libcurl` and transitively `curl`.
See https://curl.se/docs/CVE-2023-38545.html

Also, `CVE-2023-38546` has different set of affected versions.

I suggest removing mention of `CVE-2023-38546`.